Sources/Widgets/Manipulators/AbstractManipulator/index.d.ts

```diff
@@ -49,11 +49,20 @@ export interface vtkAbstractManipulator extends vtkObject {
     setUseCameraNormal(useCameraNormal: boolean): boolean;
 
     /**
+     * Processes a vtkRenderWindowInteractor event into 3D world positional info.
      * 
+     * Returns an object containing:
+     * - worldCoords: a 3D coordinate corresponding to the 2D event. Can be null.
+     * - worldDelta: a 3D position delta between the current and the previous call to handleEvent.
```

Comment:
please explain what is returned the first time it is called.

Reply:
please explain that sometimes it is important to call handleEvent "just" to have worldDelta in correct state for next call.